Cargo.toml

```diff
@@ -111,20 +111,83 @@ strata-tx-parser = { path = "crates/tx-parser" }
+reth-db = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-errors = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-ethereum-forks = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-ethereum-payload-builder = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-evm = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-evm-ethereum = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-exex = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-ipc = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-network-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-node-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-node-builder = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-node-ethereum = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-payload-builder = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0", default-features = false, features = [
+  "std",
+] }
+reth-primitives-traits = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0", default-features = false }
+reth-provider = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-revm = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-rpc = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-rpc-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-rpc-eth-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-rpc-eth-types = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0", default-features = false }
+reth-rpc-layer = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-rpc-server-types = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-rpc-types-compat = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-tasks = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-transaction-pool = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-trie = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+reth-trie-common = { git = "https://github.com/paradigmxyz/reth.git", rev = "v1.1.0" }
+
 anyhow = "1.0.86"
 arbitrary = { version = "1.3.2", features = ["derive"] }
 argh = "0.1"
 async-trait = "0.1.80"
 base64 = "0.22.1"
-bincode = "1.3.3"
+bincode = "1.3"
 bitcoin = { version = "=0.32.3", features = ["serde"] }
 bitcoind = { version = "0.36.0", features = ["26_0"] }
 borsh = { version = "1.5.0", features = ["derive"] }
 bytes = "1.6.0"
 chrono = "0.4.38"
+clap = "4"
+derive_more = { version = "1", features = ["full"] }
```

Comment:
We are not bringing `clap` to the codebase. Please use `argh` since it is less compile time and we are already using it throughout the codebase.